### PR TITLE
ipsec: fix check string against another string not the before defined variable

### PR DESCRIPTION
**Pull request by jfrede**

### CODE REVIEW

File: heartbeat/ipsec [hunk 1]
```diff
@@ -150,10 +150,14 @@ ipsec_monitor() {
 
 	ipsec status | grep "$OCF_RESKEY_tunnel" | grep -q unoriented
 	state=$?
+
+	# TODO  I think we need also to check if output of ipsec status is empty!
+	#       Then the tunnel is not running.
```
Comment (oalbrigt) [inline heartbeat/ipsec:155]: If it is empty it wont match unoriented.

Comment (jfrede) [reply heartbeat/ipsec:155]: Yes but if it is empty the check reports that the tunnel is active which is not true.

Comment (oalbrigt) [reply heartbeat/ipsec:155]: Oh.

**Comment by oalbrigt** [comment]
ok to test

**Comment by oalbrigt** [comment]
Thanks.

**Pull request merged**